### PR TITLE
Add info about permitted HTML tags for help text

### PR DESCRIPTION
**Pull request by orenyk**
### Changes

-  Help text in support of #526 

### PR Checklist:

1. Have you tagged the appropriate dev(s) for review?
2. Have you moved any Jira/Smartsheet tickets to Code Review?

### CODE REVIEW

File: app/views/service_providers/_form.html.erb [hunk 1]
```diff
@@ -350,7 +350,9 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
 
   <fieldset class="usa-fieldset-inputs custom-help-text">
     <legend>Custom help text</legend>
-    <p>You can specify help text in these different areas to alert users of important information for logging into your app</p>
+    <p>You can specify help text in these different areas to alert users of important information for logging into your app. Permitted HTML elements include the following:
+      <code>p</code>, <code>br</code>, <code>ol</code>, <code>ul</code>, <code>li</code>, <code>b</code>, <code>strong</code>, and <code>em</code>.
```
Comment (aduth) [inline app/views/service_providers/_form.html.erb:354]: I wonder if we should create a constant for the allowed tags, to share between here and `sanitize_section`.

```suggestion
      <%= safe_join(ServiceProvider::ALLOWED_HTML_ELEMENTS.map { |tag_name| content_tag(:code, tag_name) }, ', ') %>.
```

**Review by aduth — APPROVED** [review]
👍

File: app/views/service_providers/_form.html.erb (hunk 1 above)
Comment (orenyk) [reply app/views/service_providers/_form.html.erb:354]: great idea @aduth! I'll make that happen

File: app/views/service_providers/_form.html.erb [hunk 2]
```diff
@@ -350,7 +350,9 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
 
   <fieldset class="usa-fieldset-inputs custom-help-text">
     <legend>Custom help text</legend>
-    <p>You can specify help text in these different areas to alert users of important information for logging into your app</p>
+    <p>You can specify help text in these different areas to alert users of important information for logging into your app. Permitted HTML elements include the following:
+    <%= ServiceProvider::ALLOWED_HELP_TEXT_HTML_TAGS.map { |t| "<code>#{t}</code>" }.join(", ") %>.
```
Comment (aduth) [inline app/views/service_providers/_form.html.erb:354]: Looks like it's escaped:

image

Comment (orenyk) [reply app/views/service_providers/_form.html.erb:354]: 🤦 thank you for checking. I'm guessing I just need to throw an `html_safe` at the end

Comment (orenyk) [reply app/views/service_providers/_form.html.erb]: done and checked locally that it looks ok. Thanks again @aduth !

**Review by aduth — APPROVED** [review]

**Pull request merged**